### PR TITLE
[fix] Update release notes urls

### PR DESCRIPTION
**Pull request by tedsczelecki**
Update cluster url to beta-v2.0.0 in urls

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -15,11 +15,11 @@ can view active developments on GitHub at any time.
 The team is actively working on the multi-node version
 of TimescaleDB which is currently in beta. You can read more about our
 architecture and design for distributed hypertables
-[here](https://docs.timescale.com/clustering/introduction/architecture#distributed-hypertables).
+[here](https://docs.timescale.com/beta-v2.0/introduction/architecture#distributed-hypertables).
 To test out the beta version for yourself, join our #multinode-beta channel on
 [community slack](https://slack.timescale.com/) for installation details and
 follow these [setup
-instructions](https://docs.timescale.com/clustering/getting-started/scaling-out).
+instructions](https://docs.timescale.com/beta-v2.0.0/getting-started/setup-multi-node).
```
Comment (solugebefola) [inline release-notes.md:22]: The two links are different? `beta-v2.0` vs. `beta-v2.0.0`

Comment (tedsczelecki) [reply release-notes.md:22]: youre right

Comment (solugebefola) [reply release-notes.md:22]: Should be able to include as a relative link, actually, without the version?

Comment (tedsczelecki) [reply release-notes.md:22]: since these are pointing to future version, the version should be there

**Pull request merged**